sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -147,3 +152,25 @@ function buildNode(obj: any, elementName: string): Node[] {
     throw new Error(`Illegal value passed to buildObject: ${obj}`);
   }
 }
+
+export function convertAtomXmlToJson(body: string): any {
+  const dom = parser.parseFromString(body, "text/xml");
+  throwIfError(dom);
+  const result = domToObject(dom);
+  return result;
+}
+
+/**
+ * @param {object} content The content payload as it is to be serialized. It should include any root node(s).
+ */
+export function convertJsonToAtomXml(content: any): string {
+  content[Constants.XML_METADATA_MARKER] = { type: "application/xml" };
```

Comment:
Why isnt this part of the serializer?

Reply:
Since it isn't service specific  - but moved it as well, in favor of improved design later..